Task2/Project/DataControl/ViewModel/ApplicationViewModel.cs

```diff
@@ -189,49 +211,174 @@ public System.Collections.Generic.IEnumerable<string> RedoActionNames
         public RelayCommand ChangeStrokeWidth => changeStrokeWidth;
 
         public RelayCommand ChangeShapeLocation => changeShapeLocation;
+        #endregion
 
-        
 
         // METHODS
         private void NewFileMethod(object o)
         {
-            throw new System.NotImplementedException();
+            canvas.Clear();
+            OnPropertyChanged(new PropertyChangedEventArgs("dataChanged"));

```

Comment:
there is no such property as "dataChanged".
there should be a name of property that is binding to view, as we bind all property there could be any of them, but not name of field.

Reply:
Ok.